tests/test_cases/test_cocotb/test_timing_triggers.py

```diff
@@ -244,3 +247,15 @@ async def test_singleton_isinstance(dut):
     assert isinstance(NextTimeStep(), NextTimeStep)
     assert isinstance(ReadOnly(), ReadOnly)
     assert isinstance(ReadWrite(), ReadWrite)
+
+
+@cocotb.test()
+async def test_neg_timer(dut):
+    """Test negative timer values are forbidden"""
+    with assert_raises(TriggerException):
+        Timer(-42)  # no need to even `await`, constructing it is an error
+    # handle 0 special case
+    with warnings.catch_warnings(record=True) as w:
```

Comment:
```suggestion
    with warnings.catch_warnings(record=True) as w:
        warnings.simplefilter("ignore")
```

Reply:
Sorry, you were right all along - it was your test that was failing, but it's easy to fix.

Reply:
ok, thank you!

I was kinda questioning my sanity a bit^^

Reply:
Mhhh, still fail
![grafik](https://user-images.githubusercontent.com/25175069/95516223-36ef7b80-09bf-11eb-940f-1f0bde62dc1f.png)
Do we need to move the ignore filter after the assert check?